Tools/flatpak/flatpakutils.py

```diff
@@ -698,7 +698,7 @@ def setup_local_deps(self, building):
         if building:
             Console.message("Building local dependencies from %s ", src_dir)
             if self.run_in_sandbox('meson', 'compile', '-C', sandbox_build_dir, building_local_deps=True, start_sccache=False) != 0:
-                raise RuntimeError('Error while building local dependencies.')
+                raise RuntimeError('Error while building local dependencies in {}'.format(sandbox_build_dir))
```

Comment:
Since the tooling now requires Python3 you can use a f-string here.

Reply:
Cool. I like f-strings.